scripts/valhalla_build_config

```diff
@@ -35,6 +35,7 @@ config = {
     'hierarchy': True,
     'shortcuts': True,
     'include_driveways': True,
+    'include_constructions': False,
```

Comment:
configure include or exclude roads under construction

Reply:
nit: the plural `constructions` sounds odd. probably just the singular `construction` is better.